automation/DeployToHPVS.conf

```diff
@@ -0,0 +1,164 @@
+#
+# Copyright (C) 2021 IBM. All Rights Reserved.
+#
+# See the LICENSE file in the root directory
+# of this source tree for licensing information.
+#
+
+#
+# DeployToHPVS.conf
+# --------------------
+#     Configuration template file
+# 
+# Author(s)
+# ---------
+#     Lei A.B. Wang <wlwangwl@cn.ibm.com>
+#     Dan FitzGerald <danfitz@us.ibm.com>                    
```

Comment:
I would add a maintainers section or yourself as author/maintainer. We might also want to link to the github page where people can go to find the project if they received a tarball or something and want to know the current maintainer at any time in the future.